From 33f8d622559a59fa01e4099f5623af7d001de549 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timoth=C3=A9e=20Ravier?= <tim@siosm.fr>
Date: Fri, 22 Sep 2023 20:17:55 +0200
Subject: [PATCH] data/kde-portals.conf: Fallback to GTK portals for Settings

Add a fallback to the GTK portal for Settings. This makes sure that GTK
apps always get valid fonts and related settings.

See: https://bugzilla.redhat.com/show_bug.cgi?id=2240211
See: https://pagure.io/fedora-kde/SIG/issue/396
See: https://github.com/flatpak/xdg-desktop-portal/issues/1017
See: https://github.com/flatpak/xdg-desktop-portal/issues/1112
See: https://bugs.kde.org/show_bug.cgi?id=474746

BUG: 474746

See: https://invent.kde.org/plasma/xdg-desktop-portal-kde/-/merge_requests/231
(cherry picked from commit 93f96d31d14a88b45065a21616135a126131c579)
---
 data/kde-portals.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/kde-portals.conf b/data/kde-portals.conf
index 3c6c74b4..3cfc18f1 100644
--- a/data/kde-portals.conf
+++ b/data/kde-portals.conf
@@ -1,2 +1,3 @@
 [preferred]
 default=kde
+org.freedesktop.impl.portal.Settings=kde;gtk;
-- 
GitLab

